docs/source/tutorials/muon_GUI_course/the_tabs_results.rst

```diff
@@ -0,0 +1,89 @@
+.. _the_tabs_results:
+
+==================
+The Tabs - Results
+==================
+
+.. index:: The Tabs - Results
+
+.. contents:: Table of Contents
+  :local:
+  
+Results
+=======
+
+The Results tab allows the user to:
+
+* create a result(s) table
+* select which instrument log values (temp, field etc) to write out alongside the fit parameters
+* choose to write out fit information from one or several data files
+
+.. figure:: /images/thetabsresultsfig32.png
+    :align: center
+
+    Figure 32: The Results tab options.
+
+The user has chosen to create a results table. When the Output Results button is clicked, the resulting table will appear in the 
+"Workspaces" section of the main Mantid window. From here the data can be :ref:`explored <02_the_matrix_workspace>` and :ref:`plotted <03_displaying_1D_data>` as one would 
+with any data in a Mantid workspace.
+The data contained in a results table is determined by the contents of the Values and Fitting Results sections (in the example above these are empty; no data has been fitted,
+so there are no workspaces available for the Fitting Results section). 
+
+In the Values sections, the user can choose which Log Values to include in the results table, these values are data from the instrument such as run number, sample temperature etc. 
+which are taken from the workspaces in the Fitting Results section.
+
+*NB even if a workspace from the Fit Result table has not been selected (via the checkbox), the types of Log Value it contains will still be present in the Values table. 
+This does not mean they will be included in a produced results table.*
+
+The Fit Results section allows the user to choose which workspaces to use Log Values from - these can be either individual fits, or a sequential/simultaneous fits. 
+The first option in this section is the Function Name drop-down menu, selecting a certain function in this menu will show all the workspaces which have had this function fitted
+to them in the table below. By default, checking the box next to a workspace in this table means its Log Values will be present in the results table. This can be changed with the Include/Exclude option
```

Comment:
```suggestion
to them in the table below. By default, checking the box next to a workspace in this table means its Log Values will be present in the results table. This can be changed with the `Include`/`Exclude` option
```